decayAmplitude/massDependence.cc

```diff
@@ -564,6 +617,231 @@ piPiSWaveAuMorganPenningtonKachaev::piPiSWaveAuMorganPenningtonKachaev()
 }
 
 
+////////////////////////////////////////////////////////////////////////////////
+KPiSGLASS::KPiSGLASS(const double a, const double r, const double M0, const double G0, const double phiF, const double phiR, const double phiRsin,
+                     const double F, const double R, const double MMax, const std::string& tag)
+	: massDependenceImpl<KPiSGLASS>(),
+	  _a(a),
+	  _r(r),
+	  _M0(M0),
+	  _G0(G0),
+	  _phiF(phiF),
+	  _phiR(phiR),
+	  _phiRsin(phiRsin),
+	  _F(F),
+	  _R(R),
+	  _MMax(MMax),
+	  _tag(tag),
+	  _piChargedMass(0.0),
+	  _kaonChargedMass(0.0)
+{
+	particleDataTable& pdt = particleDataTable::instance();
+	const string partList[] = { "pi+", "K+" };
+	for (unsigned int i = 0; i < sizeof(partList) / sizeof(partList[0]); ++i)
```

Comment:
Maybe size_t would be better.

Reply:
`pdt.entry` does only print a warning and returns `NULL`, so you would need to make a check of that `NULL` in both cases below.